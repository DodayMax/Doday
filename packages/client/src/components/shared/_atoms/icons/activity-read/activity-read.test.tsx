import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('ActivityReadType', () => {
  it('renders correctly without props', () => {
    const icon = shallow(<Icons.ActivityReadType />);

    expect(icon).toMatchSnapshot();
  });
});
