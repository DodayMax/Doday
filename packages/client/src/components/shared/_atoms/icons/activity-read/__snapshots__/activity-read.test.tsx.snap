// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityReadType renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <defs>
          <path
            d="M5.583 12A.592.592 0 0 1 5 11.4c0-.331.261-.6.583-.6h.243C6.276 9.74 7.304 9 8.5 9a2.9 2.9 0 0 1 2.505 1.462A1.98 1.98 0 0 1 12 10.2c.364 0 .709.095.995.262A2.9 2.9 0 0 1 15.5 9c1.196 0 2.224.74 2.674 1.8h.243c.322 0 .583.269.583.6 0 .331-.261.6-.583.6 0 1.657-1.306 3-2.917 3-1.61 0-2.917-1.343-2.917-3 0-.101.005-.201.015-.3h-.015c0-.122-.23-.3-.583-.3s-.583.178-.583.3h-.015c.01.099.015.199.015.3 0 1.657-1.306 3-2.917 3-1.61 0-2.917-1.343-2.917-3zM8.5 13.8c.966 0 1.75-.806 1.75-1.8s-.784-1.8-1.75-1.8-1.75.806-1.75 1.8.784 1.8 1.75 1.8zm7 0c.966 0 1.75-.806 1.75-1.8s-.784-1.8-1.75-1.8-1.75.806-1.75 1.8.784 1.8 1.75 1.8z"
            id="a"
          />
        </defs>,
        <g
          fill="black"
          fillRule="evenodd"
        >
          <path
            d="M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z"
          />
          <use
            fillRule="nonzero"
            xlinkHref="#a"
          />
        </g>,
      ],
      "height": 20,
      "viewBox": "0 0 24 24",
      "width": 20,
      "xmlns": "http://www.w3.org/2000/svg",
      "xmlnsXlink": "http://www.w3.org/1999/xlink",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <path
            d="M5.583 12A.592.592 0 0 1 5 11.4c0-.331.261-.6.583-.6h.243C6.276 9.74 7.304 9 8.5 9a2.9 2.9 0 0 1 2.505 1.462A1.98 1.98 0 0 1 12 10.2c.364 0 .709.095.995.262A2.9 2.9 0 0 1 15.5 9c1.196 0 2.224.74 2.674 1.8h.243c.322 0 .583.269.583.6 0 .331-.261.6-.583.6 0 1.657-1.306 3-2.917 3-1.61 0-2.917-1.343-2.917-3 0-.101.005-.201.015-.3h-.015c0-.122-.23-.3-.583-.3s-.583.178-.583.3h-.015c.01.099.015.199.015.3 0 1.657-1.306 3-2.917 3-1.61 0-2.917-1.343-2.917-3zM8.5 13.8c.966 0 1.75-.806 1.75-1.8s-.784-1.8-1.75-1.8-1.75.806-1.75 1.8.784 1.8 1.75 1.8zm7 0c.966 0 1.75-.806 1.75-1.8s-.784-1.8-1.75-1.8-1.75.806-1.75 1.8.784 1.8 1.75 1.8z"
            id="a"
          />,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M5.583 12A.592.592 0 0 1 5 11.4c0-.331.261-.6.583-.6h.243C6.276 9.74 7.304 9 8.5 9a2.9 2.9 0 0 1 2.505 1.462A1.98 1.98 0 0 1 12 10.2c.364 0 .709.095.995.262A2.9 2.9 0 0 1 15.5 9c1.196 0 2.224.74 2.674 1.8h.243c.322 0 .583.269.583.6 0 .331-.261.6-.583.6 0 1.657-1.306 3-2.917 3-1.61 0-2.917-1.343-2.917-3 0-.101.005-.201.015-.3h-.015c0-.122-.23-.3-.583-.3s-.583.178-.583.3h-.015c.01.099.015.199.015.3 0 1.657-1.306 3-2.917 3-1.61 0-2.917-1.343-2.917-3zM8.5 13.8c.966 0 1.75-.806 1.75-1.8s-.784-1.8-1.75-1.8-1.75.806-1.75 1.8.784 1.8 1.75 1.8zm7 0c.966 0 1.75-.806 1.75-1.8s-.784-1.8-1.75-1.8-1.75.806-1.75 1.8.784 1.8 1.75 1.8z",
            "id": "a",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        "type": "defs",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <path
              d="M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z"
            />,
            <use
              fillRule="nonzero"
              xlinkHref="#a"
            />,
          ],
          "fill": "black",
          "fillRule": "evenodd",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "fillRule": "nonzero",
              "xlinkHref": "#a",
            },
            "ref": null,
            "rendered": null,
            "type": "use",
          },
        ],
        "type": "g",
      },
    ],
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <defs>
            <path
              d="M5.583 12A.592.592 0 0 1 5 11.4c0-.331.261-.6.583-.6h.243C6.276 9.74 7.304 9 8.5 9a2.9 2.9 0 0 1 2.505 1.462A1.98 1.98 0 0 1 12 10.2c.364 0 .709.095.995.262A2.9 2.9 0 0 1 15.5 9c1.196 0 2.224.74 2.674 1.8h.243c.322 0 .583.269.583.6 0 .331-.261.6-.583.6 0 1.657-1.306 3-2.917 3-1.61 0-2.917-1.343-2.917-3 0-.101.005-.201.015-.3h-.015c0-.122-.23-.3-.583-.3s-.583.178-.583.3h-.015c.01.099.015.199.015.3 0 1.657-1.306 3-2.917 3-1.61 0-2.917-1.343-2.917-3zM8.5 13.8c.966 0 1.75-.806 1.75-1.8s-.784-1.8-1.75-1.8-1.75.806-1.75 1.8.784 1.8 1.75 1.8zm7 0c.966 0 1.75-.806 1.75-1.8s-.784-1.8-1.75-1.8-1.75.806-1.75 1.8.784 1.8 1.75 1.8z"
              id="a"
            />
          </defs>,
          <g
            fill="black"
            fillRule="evenodd"
          >
            <path
              d="M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z"
            />
            <use
              fillRule="nonzero"
              xlinkHref="#a"
            />
          </g>,
        ],
        "height": 20,
        "viewBox": "0 0 24 24",
        "width": 20,
        "xmlns": "http://www.w3.org/2000/svg",
        "xmlnsXlink": "http://www.w3.org/1999/xlink",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <path
              d="M5.583 12A.592.592 0 0 1 5 11.4c0-.331.261-.6.583-.6h.243C6.276 9.74 7.304 9 8.5 9a2.9 2.9 0 0 1 2.505 1.462A1.98 1.98 0 0 1 12 10.2c.364 0 .709.095.995.262A2.9 2.9 0 0 1 15.5 9c1.196 0 2.224.74 2.674 1.8h.243c.322 0 .583.269.583.6 0 .331-.261.6-.583.6 0 1.657-1.306 3-2.917 3-1.61 0-2.917-1.343-2.917-3 0-.101.005-.201.015-.3h-.015c0-.122-.23-.3-.583-.3s-.583.178-.583.3h-.015c.01.099.015.199.015.3 0 1.657-1.306 3-2.917 3-1.61 0-2.917-1.343-2.917-3zM8.5 13.8c.966 0 1.75-.806 1.75-1.8s-.784-1.8-1.75-1.8-1.75.806-1.75 1.8.784 1.8 1.75 1.8zm7 0c.966 0 1.75-.806 1.75-1.8s-.784-1.8-1.75-1.8-1.75.806-1.75 1.8.784 1.8 1.75 1.8z"
              id="a"
            />,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M5.583 12A.592.592 0 0 1 5 11.4c0-.331.261-.6.583-.6h.243C6.276 9.74 7.304 9 8.5 9a2.9 2.9 0 0 1 2.505 1.462A1.98 1.98 0 0 1 12 10.2c.364 0 .709.095.995.262A2.9 2.9 0 0 1 15.5 9c1.196 0 2.224.74 2.674 1.8h.243c.322 0 .583.269.583.6 0 .331-.261.6-.583.6 0 1.657-1.306 3-2.917 3-1.61 0-2.917-1.343-2.917-3 0-.101.005-.201.015-.3h-.015c0-.122-.23-.3-.583-.3s-.583.178-.583.3h-.015c.01.099.015.199.015.3 0 1.657-1.306 3-2.917 3-1.61 0-2.917-1.343-2.917-3zM8.5 13.8c.966 0 1.75-.806 1.75-1.8s-.784-1.8-1.75-1.8-1.75.806-1.75 1.8.784 1.8 1.75 1.8zm7 0c.966 0 1.75-.806 1.75-1.8s-.784-1.8-1.75-1.8-1.75.806-1.75 1.8.784 1.8 1.75 1.8z",
              "id": "a",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          "type": "defs",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <path
                d="M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z"
              />,
              <use
                fillRule="nonzero"
                xlinkHref="#a"
              />,
            ],
            "fill": "black",
            "fillRule": "evenodd",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "d": "M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z",
              },
              "ref": null,
              "rendered": null,
              "type": "path",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "fillRule": "nonzero",
                "xlinkHref": "#a",
              },
              "ref": null,
              "rendered": null,
              "type": "use",
            },
          ],
          "type": "g",
        },
      ],
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
